app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -243,7 +243,7 @@ class ExplorationActivityTest {
 
       onView(withId(R.id.action_audio_player)).perform(click())
       onView(withId(R.id.text_input_interaction_view)).perform(ViewActions.typeText("123"), closeSoftKeyboard())
-      onView(withId(R.id.interaction_button)).perform(click())
+      onView(withId(R.id.submit_answer_button)).perform(click())
```

Comment:
There is one unused import in this file, please remove that.

Reply:
Done. Nice catch.